run_sdk_container

```diff
@@ -129,7 +129,8 @@ if [ -z "$stat" ] ; then
        -e SDK_GROUP_ID="$(id -g)" \
        --name="$name" \
        --hostname="$hostname" \
-       "${container_image_name}"
+       --entrypoint /bin/sleep \
+       "${container_image_name}" infinity
```

Comment:
I don't think this is a good idea. It will leave the container running in the background forever..

Reply:
Maybe use `/bin/true`?

Reply:
So, `sleep 10` instead? This would also work and ensure `docker exec` has enough overlap. Currently the sdk_entry script runs like 1 second which normally enough to have `docker exec` work but under high load it won't work. The container has to run for `docker exec` to succeed. Since we have the cleanup, I thought inifinity is not that bad and the user at least sees the container in `docker ps` while a stopped container is only shown with `docker ps -a`.

Reply:
I'll try to keep the same behavior by running `bash -l`

Reply:
That works